ci/autoresearch/phases.py

```
@@ -1285,6 +1327,217 @@ async def _run_simd_tests(ctx: RunContext) -> int:
+        echo_ok = result_token in accumulated
+        # Look for the FL_DBG line from fl::Remote
+        dbg_token = b"Stored request ID for echo"
+        log_ok = dbg_token in accumulated
+        # Look for the REMOTE: prefix proving Serial.println via the sink
+        remote_ok = b"REMOTE: " in accumulated
+
+        passed = echo_ok and remote_ok
+        if passed:
+            print(f"{Fore.GREEN}BRING-UP TEST PASSED{Style.RESET_ALL}")
+            print(
+                f"   ✅ Serial.println — REMOTE: response received"
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)